pkg/validations/common_bucket_validations.go

```diff
@@ -7,8 +7,38 @@ import (
 	"github.com/noobaa/noobaa-operator/v5/pkg/nb"
 	"github.com/noobaa/noobaa-operator/v5/pkg/system"
 	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+	nbv1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
 )
 
+// ValidateNSFSAccountConfig validates that the provided NSFS config is valid
+func ValidateNSFSAccountConfig(NSFSConfig string) error {
+	log := util.Logger()
+
+	if NSFSConfig == "" {
+		return nil
+	}
+
+	var configObj nbv1.AccountNsfsConfig
+
+	err := json.Unmarshal([]byte(NSFSConfig), &configObj)
+	if err != nil {
+		return fmt.Errorf("failed to parse NSFS config %q: %v", NSFSConfig, err)
+	}
+
+	log.Infof("Validating NSFS config: %+v", NSFSConfig)
+	// UID validation
+	if configObj.UID < 0 {
+		return fmt.Errorf("UID must be a whole positive number")
+	}
+
+	// GID validation
+	if configObj.GID < 0 {
+		return fmt.Errorf("GID must be a whole positive number")
+	}
+
+	return nil
+}
+
```

Comment:
Please add unit tests

Reply:
As we discussed on Slack - 
validation tests to ValidateNSFSAccountConfig()
we have integration tests also (cli flow)